perfaware/sim86/shared/Sim8086.cpp

```diff
@@ -128,6 +135,19 @@ static u16 GetRightOperandValue(const instruction_operand& Source, const s16* Re
         {
             RightOperandValue = static_cast<u16>(Source.Immediate.Value);
         } break;
+        case Operand_Memory:
+        {
+            size_t EffectiveAddress = ComputeEffectiveAddress(Source, Registers);
+            if (Source.Register.Count == 2) // word value
+            {
+                u16* DestPointer = reinterpret_cast<u16*>(&Memory[EffectiveAddress]);
```

Comment:
Why is this called `DestPointer`?

Reply:
Because I switched context after copy-pasting and forgot about it
Fixed
8cf545beb55a74b50471fa417c08318df9ea2a4c